tendermint/validator.go

```diff
@@ -3,48 +3,58 @@ package tendermint
 import (
 	cryptoTypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	"github.com/cosmos/cosmos-sdk/types"
+	slashingTypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingTypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	"github.com/cosmos/relayer/relayer"
+
 	"github.com/persistenceOne/persistenceBridge/application/db"
 	"github.com/persistenceOne/persistenceBridge/utilities/logging"
 )
 
-//Map to keep track of missed blocks during previous CheckValidators() call
-var missedBlockCounterForValidator = make(map[string]int64)
```

Comment:
@arhamchordia I have a question here. I believe we should reset the map for every new block, don't we?

Reply:
@JekaMas Cannot reset it, as we are sending notification only when there is an increment in missed blocks (after an interval of say x number of blocks). 
If one does not miss any blocks in that interval then the missed block counter is set to the value of missedBlocksCounter (which is zero when no blocks have been missed).

Reply:
got it. Thank you!